pubtools/_quay/command_executor.py

```diff
@@ -46,7 +47,7 @@ def skopeo_login(self, username=None, password=None):
 
         cmd_login = (
             "skopeo login --authfile $HOME/.docker/config.json -u {0} --password-stdin quay.io"
-        ).format(username)
+        ).format(pipes.quote(username))
```

Comment:
I wonder, is this safe for Python 3? The tests passed, but the documentation makes it seem like it's deprecated. Could you pls use the 'six' module for it instead?

Reply:
Yeah, fixed.

Reply:
There's probably a more straightforward way to do this. We're quoting a string and substituting it into another string, then shlex.splitting it before passing it to subprocess.Popen. Maybe it should just be a list of strings to start with?

Reply:
The reason for using quote is that remote execution is done with 'paramiko' library, which doesn't support splitting the input (AFAIK)

Reply:
Ah, ok.